src/eolab/rastertools/cli/__init__.py

```diff
@@ -69,4 +69,4 @@ def with_bands_arguments(parser):
         '--all',
         dest="all_bands",
         action="store_true",
-        help="Compute all bands")
+        help="Compute all bands")
```

Comment:
Not sure why there is a modification here.

Reply:
I accidentaly deleted this line on a previous commit. I writed it back after noticing my mistake

Reply:
Sure, but this should no showup in the final diffs, maybe there a different in white space or newlines?